### ISSUE TITLE
SDK Hangs/Lags on specific Blender Models

### ISSUE THREAD

**Issue by MeFisto94** [jMonkeyEngine/sdk#56]
Hey Guys,
Hey @jmekaelthas 

I know this is actually a SDK Problem and nothing with the Blender Importer but maybe Kaelthas can Imagine what's going on better than us.

I have a model file which works fine and really fast on the 3.0 SDK but when Importing on 3.1 (alpha5), I had to kill the SDK because after 7 minutes of loading it stopped logging triangulation errors and instead did nothing but that at 100% CPU Load :smile: 

I then tried to write a simple application, which also took long (3-4 minutes), spammed the log but finally managed to convert the file:

```
Spatial s = assetManager.loadModel("Blender_SimpleTest/Raige_Two_packed.blend");
BinaryExporter exporter = new BinaryExporter();
exporter.save(s, new File("test.j3o"));
```

Now the Problem is that this .j3o is nearly un-editable because it's quite laggy where as you get ~350 FPS when simply extending the upper code by rootNode.attachChild(s);

What is it that this model has to bug out sdk's. Can you guys even reproduce that?
[Raige_Two_packed.blend.zip](https://github.com/jMonkeyEngine/sdk/files/222719/Raige_Two_packed.blend.zip)

Note: Just because the Model is uploaded here doesn't give anybody the permission to use it in their games or something ;)

**Comment by Dokthar** [comment]
you should send triangle mesh to jme, so in blender add the modifier triangulate and then apply all your modifiers.

**Comment by jmekaelthas** [comment]
I agree that applying triangulation would help. But it is just a workaround. I downloaded the model and I'll try to reproduce it.

Definitely I'll need to reduce the amount of warning logs during triangulation as there might be thousands of faces that are not properly triangulated.

**Comment by Dokthar** [comment]
even in blender the triangulation looks weird, and there also other modifiers (than triangulation) which i believe must be applied before import.

**Comment by MeFisto94** [comment]
Yeah that could be a solution but in comparison to 3.0 it is way slower and in comparison to a standalone App it's twice as slow, so we should at least look into it so maybe we have a clue.

It could simply be that the logger is slowing it down that much, because that is new in 3.1

**Comment by jmekaelthas** [comment]
Hey @MeFisto94 
I have made a change to the importer about the huge amount of warning logs during triangulation.
Now, if any problems occur for a mesh it will be logged only once.

Please let me know if you can see the model in SDK after the change.

But despite the logging problem I saw another. The model was semi transparent after loading. So I think I'll work on it a little bit longer.

**Comment by MeFisto94** [comment]
Now it works like a charm. It seems like we were too fast and hence filled the Output Buffer which caused the SDK to lock up.

Appart from the transparency problem it seems the SDK doesn't display the Animations any longer but I need to do some investigation on this before treating it as a bug.

**Comment by jmekaelthas** [comment]
Today I managed to fix the transparency problem. Let me know if it works better now ;)

**Comment by MeFisto94** [comment]
Yep, this solved the see-through for solids but there still is something:
The hair, eyebrows etc (so things actually using Transparency) appear whiteish.
I had that problem back then but don't remember how I fixed it (Something like turning alpha channels down and other stuff up and idk).

Here is a reference picture how it seems to look in xbuf (thanks Ali for that)
![image](https://jme-hub-cdn-jmonkeyengineor.netdna-ssl.com/uploads/default/original/2X/6/62fc9c6bd98589d563bf44ef72f5b6e067529bd2.png)

But thanks for the quick fixes :)
